File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/comment/exception/CommentNotMatchEventException.java
package band.gosrock.domain.domains.comment.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class CommentNotMatchEventException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new CommentNotMatchEventException();

    private CommentNotMatchEventException() {
        super(CommentErrorCode.COMMENT_NOT_MATCH_EVENT);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/comment/exception/CommentNotMatchEventException.kt
package band.gosrock.domain.domains.comment.exception

import band.gosrock.common.exception.DuDoongCodeException

class CommentNotMatchEventException : DuDoongCodeException(CommentErrorCode.COMMENT_NOT_MATCH_EVENT) {
    companion object {
        val EXCEPTION: DuDoongCodeException = CommentNotMatchEventException()
    }
}
